cli/client/create_lease.go

```diff
@@ -0,0 +1,39 @@
+package client
```

Comment:
not-blocking-for-this-PR-question: thoughts on moving this package to `./client`?